From: Jan Engelhardt <jengelh@inai.de>
Date: 2020-06-22 21:16:10.343429323 +0200

Make use of the clzma library in openSUSE.
[openSUSE-specific patch. I was under the impression that Debian and/or Ubuntu shipped
lzma-sdk-dev too, but, upon checking, do not do so (anymore) as of 2024-11-27.]
---
 thirdparty/CMakeLists.txt |    3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

Index: SLADE-3.2.7/thirdparty/CMakeLists.txt
===================================================================
--- SLADE-3.2.7.orig/thirdparty/CMakeLists.txt
+++ SLADE-3.2.7/thirdparty/CMakeLists.txt
@@ -17,7 +17,6 @@ endif()
 set(EXTERNAL_SOURCES
 )
 file(GLOB_RECURSE EXTERNAL_SOURCES
-	lzma/C/LzmaDec.c
 	glad/src/*.c
 	${DUMB_SOURCES}
 	${SLADE_HEADERS}
@@ -41,7 +40,7 @@ add_subdirectory(lunasvg/3rdparty/softwa
 add_subdirectory(lunasvg/3rdparty/plutovg)
 
 add_library(external STATIC ${EXTERNAL_SOURCES})
-target_link_libraries(external ${ZLIB_LIBRARY} lunasvg fmt::fmt ${CMAKE_DL_LIBS})
+target_link_libraries(external ${ZLIB_LIBRARY} -lclzma lunasvg fmt::fmt ${CMAKE_DL_LIBS})
 set(EXTERNAL_LIBRARIES external PARENT_SCOPE)
 
 if(USE_SYSTEM_DUMB)
